mflike/mflike.py

```diff
@@ -379,18 +390,50 @@ def get_foreground_model(fg_params, fg_model,
         {"ell": ell, "ell_0": ell_0})
     model["tt", "cibp"] = fg_params["a_p"] * cibp(
         {"nu": frequencies, "nu_0": nu_0,
-         "temp": fg_params["T_d"], "beta": fg_params["beta_p"]},
-        {"ell": ell, "ell_0": ell_0, "alpha": 2})
+        "temp": fg_params["T_d"], "beta": fg_params["beta_c"]},
```

Comment:
here I'm a bit lost because we were using `beta_p` for `cibp` and `beta_c` for `cibc` component. I see that `alpha` also change and the multipole values are also different. Maybe these are all consistents given the normalization but I'm just asking in case

Reply:
In the ACT foreground model  beta_c = beta_p. alpha changes consistently with the new normalisation